docs/core/02-how-to-guides/01-plugin-development/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Plugin Development
+---
+
+# Developing your Plugin
+
+AragonOS makes it easy for you to write, maintain, and distribute your own plugin.
+
+In the following, you find guides explaining how to develop and publish plugin on the Aragon Marketplace.
```

Comment:
```suggestion
In the following, you find guides explaining how to develop and publish plugin on the Aragon UI.
```

Reply:
I wrote
> In the following, you find guides explaining how to develop and publish plugin on Aragon's repository.
because the smart contracts folder is not related to any UI